azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/redux/action/LocalParticipantAction.kt

```diff
@@ -9,6 +9,7 @@ import com.azure.android.communication.ui.calling.redux.state.AudioOperationalSt
 import com.azure.android.communication.ui.calling.redux.state.CameraDeviceSelectionStatus
 
 internal sealed class LocalParticipantAction : Action {
+    class DeviceManagerFetchFailed(val error: CallCompositeError) : LocalParticipantAction()
```

Comment:
would it be possible to indicate the cause of Fetch failed? like DeviceErrorType

Reply:
It is documented in the CallCompositeErrorCode class about the unknowError code